src/_tutorials/server/cmdline.md

```diff
@@ -97,36 +97,42 @@ You might notice that dcat depends on a package named **args**.
 To get the args package, use the
 [pub package manager](/guides/packages).
 
-A real app has tests, license files, and so on, in a file hierarchy
-such as the one in the [Dart command-line app template.][]
-But for this first app, let's do the minimum necessary to get the code to run:
+A real app has tests, license files, dependency files, examples, and so on.
+For the first app though, we can easily create only what is necessary
+with the [`dart create`](/tools/dart-tool) command.
 
-[Dart command-line app template.]: https://github.com/dart-lang/stagehand/tree/master/templates/console-full
+1. Inside a directory, create the dcat app with the dart tool.
+   
+   ```terminal
+   $ dart create dcat
+   ```
+   
+2. Change to the created directory.
 
-1. Create a directory named `dcat`, and change to that directory.
-2. Inside `dcat`, create a file named `dcat.dart`
-   and copy the preceding code into it.
-3. Inside `dcat`, create a file named `pubspec.yaml`
-   with the following code:
+   ```terminal
+   $ cd dcat
+   ```
+   
+3. Inside `dcat`, open the file named `pubspec.yaml`, 
```

Comment:
We could instead use `dart pub add args`, and just explain that it edits the pubspec.yaml file and gets the package you added. What do you think?

Reply:
I think using the command is a great idea. It was my original goal with switching to `dart create` and didn't even think about using the pub commands. Thanks!